src/Types/TypeLoader.php

```diff
@@ -199,7 +200,7 @@ protected function createSpecial(string $name): Type
             return new EdgeColumnMappingType($name, $this);
```

Comment:
fallen nicht hier die zweiten Attribute weg? same for PaginatorType, usw

Reply:
> Passen die Funktions Signaturen?

Was genau sollte nicht passen damit?

> fallen nicht hier die zweiten Attribute weg? same for PaginatorType, usw

In einem zukünftigen PR fällt irgendwann diese ganze TypeLoader Klasse weg. Die ist als ganzes deprecated, und soll komplett entfernt werden.

Ich schätze mal mit "die zweiten Attribute" ist "der zweite Constructor-Parameter" gemeint? Ich hab die jetzt mal entfernt wo nicht mehr notwendig... für manche Constructor sind sie noch notwendig, weil der TypeLoader dort noch nicht entfernt ist. Kommt alles in einem späteren PR.